server/conn.go

```diff
@@ -1076,8 +1078,12 @@ func (cc *clientConn) handleLoadData(ctx context.Context, loadDataInfo *executor
 
 	var shouldBreak bool
 	var prevData, curData []byte
-	// TODO: Make the loadDataRowCnt settable.
-	loadDataInfo.SetMaxRowsInBatch(defaultLoadDataBatchCnt)
+
+	if defaultLoadDataBatchCnt != initialDefaultLoadDataBatchCnt {
```

Comment:
why not use `DMLBatchSize` directly?

Reply:
@jackysp https://github.com/pingcap/tidb/blob/master/server/tidb_test.go#L75 changed `defaultLoadDataBatchCnt` for testing purpose, so I keep `defaultLoadDataBatchCnt` a high priority to make the test happy.